### PR TITLE
Tools for schema bundling and Draft-07 conversion

### PR DESCRIPTION
**Pull request by letmaik**
- `tools/bundle_schema.py`: bundle all referenced schemas into a single file, starting from a root schema
- `tools/downgrade_schema_to_draft07.py`: downgrades a bundled schema to Draft-07, useful since many libraries still don't support higher versions

`test/validator.py` has been moved to `tools/validator.py` with a minimal CLI to validate a CovJSON document.

Tests have been changed to run schema validation both against the "native" schemas and the Draft-07 downgraded schemas.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -26,4 +26,4 @@ jobs:
       run: pip install -r requirements.txt
       
     - name: Run tests
-      run: pytest --exhaustive
+      run: python -m pytest --exhaustive
```
Comment (jonblower) [inline .github/workflows/ci.yml:29]: Out of interest, what's the difference in behaviour here?

File: test/conftest.py [hunk 2]
```diff
@@ -1,13 +1,54 @@
...
-
 from pathlib import Path
 import json
 from copy import deepcopy
 import pytest
+import jsonschema
+
+import tools.validator as validator_
+from tools.bundle_schema import bundle_schema
+from tools.downgrade_schema_to_draft07 import downgrade_schema_to_draft07
```
Comment (jonblower) [inline test/conftest.py:10]: Why might we need to downgrade to draft07?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (letmaik) [reply .github/workflows/ci.yml:29]: `-m` always add the current folder to PYTHONPATH, which I needed to import modules from tools/

File: test/conftest.py (hunk 2 above)
Comment (letmaik) [reply test/conftest.py:10]: Most libraries are still stuck with draft07, so for widest support this is unfortunately a requirement. Someone on the OGC repo also mentioned that, for some Java piece that supports CovJSON.

Comment (jonblower) [reply test/conftest.py:10]: Are there any issues to be aware of connected with the downgrading? Any features that will be missing, or constructs we should avoid?

**Review by jonblower — APPROVED** [review]

File: test/conftest.py (hunk 2 above)
Comment (letmaik) [reply test/conftest.py:10]: Running the tests against both the downgraded schemas and the native ones makes sure that both are consistent. There may be some constructs we don't use yet that can't be downgraded, but we'll notice it once it happens.

**Pull request merged**